spec/jobs/scheduled/enqueue_reminders_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe Jobs::EnqueueReminders do
+  let(:user) { Fabricate(:user) }
+
+  after do
+    monthly_frequency = 43200
+    SiteSetting.remind_assigns_frequency = monthly_frequency
```

Comment:
There is no need to reset site settings after each test because we use an in memory site settings provider that is automatically cleared after each test.